README.md

```diff
@@ -141,7 +141,195 @@ Here is a table of algorithms, the figure, name of the algorithm in the book and
+| 14.9   | Enumeration-Ask                   | `enumeration_ask`             | [`probability.py`][probability] | Done | Included |
+| 14.11  | Elimination-Ask                   | `elimination_ask`             | [`probability.py`][probability] | Done | Included |
+| 14.13  | Prior-Sample                      | `prior_sample`                | [`probability.py`][probability] |      | Included |
+| 14.14  | Rejection-Sampling                | `rejection_sampling`          | [`probability.py`][probability] | Done | Included |
+| 14.15  | Likelihood-Weighting              | `likelihood_weighting`        | [`probability.py`][probability] | Done | Included |
+| 14.16  | Gibbs-Ask                         | `gibbs_ask`                   | [`probability.py`][probability] | Done | Included |
+| 15.4   | Forward-Backward                  | `forward_backward`            | [`probability.py`][probability] | Done |          |
+| 15.6   | Fixed-Lag-Smoothing               | `fixed_lag_smoothing`         | [`probability.py`][probability] | Done |          |
+| 15.17  | Particle-Filtering                | `particle_filtering`          | [`probability.py`][probability] | Done |          |
+| 16.9   | Information-Gathering-Agent       |                               |                                 |      |          |
+| 17.4   | Value-Iteration                   | `value_iteration`             | [`mdp.py`][mdp]                 | Done | Included |
+| 17.7   | Policy-Iteration                  | `policy_iteration`            | [`mdp.py`][mdp]                 | Done | Included |
+| 17.9   | POMDP-Value-Iteration             |                               |                                 |      |          |
+| 18.5   | Decision-Tree-Learning            | `DecisionTreeLearner`         | [`learning.py`][learning]       | Done | Included |
+| 18.8   | Cross-Validation                  | `cross_validation`            | [`learning.py`][learning]       |      |          |
+| 18.11  | Decision-List-Learning            | `DecisionListLearner`         | [`learning.py`][learning]\*     |      |          |
+| 18.24  | Back-Prop-Learning                | `BackPropagationLearner`      | [`learning.py`][learning]       | Done | Included |
+| 18.34  | AdaBoost                          | `AdaBoost`                    | [`learning.py`][learning]       | Done | Included |
+| 19.2   | Current-Best-Learning             | `current_best_learning`       | [`knowledge.py`](knowledge.py)  | Done | Included |
+| 19.3   | Version-Space-Learning            | `version_space_learning`      | [`knowledge.py`](knowledge.py)  | Done | Included |
+| 19.8   | Minimal-Consistent-Det            | `minimal_consistent_det`      | [`knowledge.py`](knowledge.py)  | Done |          |
+| 19.12  | FOIL                              | `FOIL_container`              | [`knowledge.py`](knowledge.py)  | Done | Included |
```

Comment:
A part of the README has been duplicated by mistake

Reply:
Thanks @ad71